src/without/index.ts

```diff
@@ -0,0 +1,3 @@
+export default function without<ItemType>(arr: ItemType[], item: ItemType) {
```

Comment:
lodash version supports multiple items. https://lodash.com/docs/4.17.15#without

Reply:
There's also [`difference`](https://lodash.com/docs/4.17.15#difference) that does exactly the same if I'm not mistaken. I vote for simplicity.

Reply:
We agreed to rename the function to `remove`